### DISCUSSION TITLE
Add weights to item-info

### DISCUSSION THREAD

**Discussion by stefaz** [Lailloken/Exile-UI#276] [Ideas & Feedback]
Hi,

first things first: thanks for your continued development of this tool and making it better with every release. I'm a long time user and with the new 1.50 version this thing works smooth as never before for me - huge thumbs-up!

with that being said, I have a feature request for the item-info module: It would be great if you could display the affix weights not only for the base jewels but for all items (in addition to the tiers). I think even with the tiering system, the weights are useful information when judging the value of an item.

Thanks!

**Comment by Lailloken** [comment]
Ah yes, I remembered your name from somewhere and just checked again. It was also related to the item-info tooltip. Thanks for the kind words. Btw, 1.50 now behaves as you requested once: clicking into the client closes it.
I'll think about adding weights as an option, but I'd have to figure out how to display them since they're up to 4 digits in length, which currently wouldn't fit well.
